### DISCUSSION TITLE
Support for Delta Pro Ultra?

### DISCUSSION THREAD

**Discussion by dfstarr** [tolwi/hassio-ecoflow-cloud#218] [General]
Hi,

Is there currently any support for the Delta Pro Ultra? If not, will there be in the future?

Thanks.

**Comment by GadgetTech** [comment]
I too would like to know.  If there is anything either of us could do to help, please let me know.  I tried to add under diagnostic but nothing populated.

**Comment by RyanWor** [comment]
Just FYI I opened a feature request earlier this year for the DPU. I unfortunately did not have spare time to help test but it looks like some progress was made. Hoping support can be added before long. 

https://github.com/tolwi/hassio-ecoflow-cloud/issues/193
